### DISCUSSION TITLE
After updating to 1.5, assemblies/tracks not loading

### DISCUSSION THREAD

**Discussion by azatian** [GMOD/jbrowse-components#2520] [Q&A] [answered]
We updated our JBrowse 2 to the latest version. Our tracks and assemblies were loaded manually on our server and were loading perfectly fine until the update. See issue below. What seems to be problem
![Screen Shot 2021-11-15 at 1 13 52 PM](https://user-images.githubusercontent.com/9220290/141841600-11149f62-9e75-4881-a498-091bd49c73cc.png)
?

**Comment by cmdcolin** [comment, marked as answer]
Hmm, i am not sure what would cause this. 

Could you clear cache in the browser, and maybe check the developer tools in chrome and see if there is any console.log output? Could also check in the network tab also perhaps to see if there is any errors in requesting data files or js files?

**Comment by azatian** [comment]
Thank you for your help! It looks like there is an error when a request is made to the following .js file:

![Screen Shot 2021-11-15 at 2 40 41 PM](https://user-images.githubusercontent.com/9220290/141851093-ad63f2c1-6a72-4ecd-97bc-64b7932ec5bf.png)

**Reply by azatian** [reply]
And my jbrowse2 directory looks like this:
12.b54801f9102a3d908696.worker.js.LICENSE.txt  
7.b54801f9102a3d908696.worker.js.LICENSE.txt  
index.html
13.b54801f9102a3d908696.worker.js.LICENSE.txt  
asset-manifest.json                                                   
manifest.json
15.b54801f9102a3d908696.worker.js.LICENSE.txt  
b54801f9102a3d908696.worker.js.LICENSE.txt    
robots.txt
16.b54801f9102a3d908696.worker.js.LICENSE.txt  
config.json                                                                  
static
37.b54801f9102a3d908696.worker.js.LICENSE.txt  
data                                                                             
test_data
4.b54801f9102a3d908696.worker.js.LICENSE.txt   
favicon.ico                                                                   
trix
5c9f1639441ae475a829.worker.js.LICENSE.txt        
version.txt

**Reply by cmdcolin** [reply]
interesting, thanks for the details. If the file "b54801f9102a3d908696.worker.js" is giving a 404 and is not on the filesystem this might indicate the update might have gone wrong

I think v1.5.0 has the file "b54801f9102a3d908696.worker.js", you could safely re-try upgrading to this version

    jbrowse upgrade --tag v1.5.0 --out /path/to/your/instance


Then ls /path/to/your/instance again and see if b54801f9102a3d908696.worker.js is present (after running this in my version, it is present)

Or you could get the latest v1.5.1, has some nice bugfixes over v1.5.0 :)

    jbrowse upgrade --out /path/to/your/instance

alternatively you can download the zips from our github release page. that is all the jbrowse upgrade does is download the zip and unpack it

**Reply by azatian** [reply]
Yup, that did it for me. Thanks again for your help in resolving this painlessly.
